DrupalExtended/ruleset.xml

```diff
@@ -0,0 +1,175 @@
+    <exclude name="SlevomatCodingStandard.Classes.RequireConstructorPropertyPromotion"/>
+    <!-- Forces for each method, not only constructors. -->
+    <exclude name="SlevomatCodingStandard.Classes.RequireMultiLineMethodSignature"/>
+    <!-- Not very friendly for constructor property promotion with a single param. -->
+    <exclude name="SlevomatCodingStandard.Classes.RequireSingleLineMethodSignature"/>
+    <exclude name="SlevomatCodingStandard.Classes.SuperfluousAbstractClassNaming"/>
+    <exclude name="SlevomatCodingStandard.Classes.SuperfluousErrorNaming"/>
+    <exclude name="SlevomatCodingStandard.Classes.SuperfluousExceptionNaming"/>
+    <exclude name="SlevomatCodingStandard.Classes.SuperfluousInterfaceNaming"/>
+    <exclude name="SlevomatCodingStandard.Classes.SuperfluousTraitNaming"/>
+    <exclude name="SlevomatCodingStandard.Classes.TraitUseDeclaration"/>
+    <exclude name="SlevomatCodingStandard.Classes.TraitUseSpacing"/>
+    <exclude name="SlevomatCodingStandard.Commenting.DeprecatedAnnotationDeclaration"/>
+    <exclude name="SlevomatCodingStandard.Commenting.DisallowCommentAfterCode"/>
+    <!-- 'Drupal.Commenting.DocComment.SpacingBeforeTags' should cover it. -->
+    <exclude name="SlevomatCodingStandard.Commenting.DocCommentSpacing"/>
+    <exclude name="SlevomatCodingStandard.Commenting.ForbiddenAnnotations"/>
+    <exclude name="SlevomatCodingStandard.Commenting.ForbiddenComments"/>
+    <exclude name="SlevomatCodingStandard.Commenting.InlineDocCommentDeclaration"/>
+    <exclude name="SlevomatCodingStandard.Commenting.RequireOneLineDocComment"/>
+    <exclude name="SlevomatCodingStandard.Commenting.RequireOneLinePropertyDocComment"/>
+    <!-- Conflicts with Drupal coding standards. -->
+    <exclude name="SlevomatCodingStandard.Commenting.UselessInheritDocComment"/>
+    <exclude name="SlevomatCodingStandard.Complexity.Cognitive"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.AssignmentInCondition"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.DisallowNullSafeObjectOperator"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.DisallowShortTernaryOperator"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.DisallowYodaComparison"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.EarlyExit"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.JumpStatementsSpacing"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.NewWithParentheses"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.NewWithoutParentheses"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.RequireMultiLineCondition"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.RequireMultiLineTernaryOperator"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.RequireSingleLineCondition"/>
```

Comment:
It might be handy, because Drupal coding standards implies that this is required.